scripts/layerdiff.js

```diff
@@ -76,10 +80,9 @@ const getId = function (f) {
 };
 
 const toFeaturePoint = function (geom) {
-  const { x, y } = geom.getCoordinates()[0];
```

Comment:
this method was not available on the updated API, using `getX` and `getY` instead.